client/src/pages/ChatPage.js

```diff
@@ -32,13 +32,16 @@ class ChatPage extends Component {
             return <p>no messages yet! {JSON.stringify(this.props)}</p>
         }
     }
-
     render() {
         return (
             <Container fluid className="vh-100 text-center" >
                 <Row className="h-10 bg-dark text-light sticky-top">
                     <Col>
-                        <h3>Chatr</h3>
+                        <h1>Chatr</h1>
```

Comment:
These are on different lines right now, you should put them in separate columns i think. Also, if this logout button works you should add back in the localstorage loggedIn thing so you don't log out just by refreshing

Reply:
will address in future commit